clients/src/test/java/org/apache/kafka/common/memory/GarbageCollectedMemoryPoolTest.java

```diff
@@ -28,63 +28,63 @@
 
     @Test(expected = IllegalArgumentException.class)
     public void testZeroSize() throws Exception {
-        new GarbageCollectedMemoryPool(0, 7, true, null);
+        new GarbageCollectedMemoryPool(0, 7, true, null, null);
```

Comment:
all the added nulls in this file would not be requires is the original signature was maintained